### ISSUE TITLE
xc can't find iraf.h during build

### ISSUE THREAD

**Issue by olebole** [iraf/iraf-v216#28]
When building from source, I get a number of errors like
```
xc -c -w -/g -/m64 [...] aclrc.c [...]
[...]
aclrc.c:6:18: fatal error: iraf.h: No such file or directory
 #include <iraf.h>
                  ^
compilation terminated.
```

**Comment by RobSteele49** [comment]
Could this in fact be related to the mkpkg.inc file I found that defines *FLAG values? @joequant @zcsteele

**Comment by iraf** [comment]
No

**Comment by RobSteele49** [comment]
Could @iraf elaborate on the "No". My idea was just a guess - but if you 'know' the fix could you share? @zcsteele and I are having similar problems and would like to know the fix that could then be pulled into the iraf-v216 master. I'm sure others would like to know the fix as well.

**Comment by olebole** [comment]
This problem is fixed by #29; at least if @iraf merges it.
